packages/react/src/components/badge/tests/Badge.spec.tsx

```diff
@@ -1,20 +1,21 @@
+import {LockSize16Px} from '@coveord/plasma-react-icons';
 import {render, screen, within} from '@test-utils';
 import * as React from 'react';
 
 import {Badge} from '../Badge';
 
 describe('Badge', () => {
     it('renders a badge', () => {
-        render(<Badge label="label" icon="lock" />);
+        render(<Badge label="label" icon={LockSize16Px} />);
 
         const badge = screen.getByLabelText('badge');
         expect(badge).toBeInTheDocument();
         expect(within(badge).getByText('label')).toBeInTheDocument();
-        expect(within(badge).getByRole('img', {name: 'lock icon'})).toBeInTheDocument();

```

Comment:
Should we keep the `icon` in the label? That will be a lot of breaking changes otherwise (thinking about the admin here)

Reply:
I would go with what is cleaner, we will have a lot of changes anyway